helm_deploy/laa-dces-drc-integration/templates/ingress.yaml

```diff
@@ -23,39 +26,49 @@ spec:
   tls:
     {{- range .Values.ingress.tls }}
     - hosts:
-        {{- range .hosts }}
+        {{- range .host }}
```

Comment:
So `hosts` was an array of objects before (each array entry in the `hosts` array was an object with a string `host` and a string array named `paths`).

You should probably still have the old CP wildcard names available as non-MTLS (as well as the `mon....` hostname) so we can move the monitoring over to the new hostname gradually (rather than break all the existing monitoring - as this will).

Reply:
I have changed the yaml files to move all arrays into values files and iterate over them in ingress. Ready for review.

Reply:
Sorry, just noticed this is now an array of string arrays, same as the `api` and `mon` objects, rather than an array of objects.

```yaml
    mon:      ### `mon` is an object because the sub-items indented from it don't use bullets
      hosts:  ### `hosts` is an array of objects because the sub-items indented from it do use bullets (the first two objects are simple objects with one key in them)
        - host: myhostname1
        - host: myhostname2
        - host: myhostname3
          extra: myextra3 ### the third object in the array has two keys

Reply:
Please check again, I think it's there now...